docs/docusaurus/docs/get_support.md

```diff
@@ -0,0 +1,57 @@
+
+## Try solving your issue first
+
+The following are some recommendations for resolving issues on your own. Resolving issues on your own builds your own knowledge and provides you with an opportunity to share that knowledge with the community. 
+
+### Search the GX documentation
+
+Search the docs you're using currently for an answer to your issue or question. If you're new to GX Core, try the [Quickstart](tutorials/quickstart/quickstart.md) or one of the specific getting started guides such as [Get started with Great Expectations and Databricks](tutorials/getting_started/how_to_use_great_expectations_in_databricks.md), or [Get started with Great Expectations and SQL](tutorials/getting_started/how_to_use_great_expectations_with_sql.md).  
+
+### Search with your favorite browser
+
+It's very likely that someone, somewhere has previously encountered or solved the issue you're currently experiencing. Perform a keyword search in your favorite browser to discover resources outside GX documentation. 
+
+### Try it out
+
+If you're wondering what might happen when you implement a solution, try it out! If you attempt a solution in your development environment, users are unaffected and unaware of what you're up to. Sneaky. 
+
+## Ask the right questions and provide the right information
+
+Taking the time to formulate your questions properly and providing all the relevant background information can save time and improve response times. The following are some recommendations that will help the GX Developer Advocates (DAs) help you. 
+
+### Include all the relevant information
+
+Tell the GX DAs exactly what issue you're experiencing, or the problem you're trying to solve. When asking questions, you should provide the following:
+
+- The complete error log or logs.
+
+- The relevant code you're trying to run.
+
+- The version of GX Core you're using.
+
+- A detailed explanation of what you're trying to resolve including what attempts you've made on your own to date.
+
+### Don't post in multiple channels and use threads
+
+When posting questions or asking for help in one of the GX Slack channels, don't post in multiple channels. Instead, select a single, relevant channel and post your message there. When responding to a post, use threads. If you need to post large code blocks, post them in a thread to avoid overwhelming the channel.
+
+### Don't tag single or multiple users
+
+If you've developed a rapport with a specific GX DA, or a community member, don't tag them in Slack posts. You never know who is available or who has the knowledge to solve your issue. Tagging specific users can limit the scope of the response. Also, the community is typically very responsive and helpful. If you don't get a response within a reasonable time, review your post and make sure you've framed your questions correctly and provided the necessary background information. If you don't get a response, or it's unhelpful, try an alternate medium to resolve your issue.
```

Comment:
Should this read "Even you've previously developed a rapport with a specific GX DA, or a community member, don't tag them in new Slack posts."?

Reply:
@rachhouse the edit was implemented as requested.